Source/ThirdParty/skia/CMakeLists.txt

```diff
@@ -0,0 +1,1019 @@
+    src/core/SkPictureData.cpp
+    src/core/SkPictureFlat.cpp
+    src/core/SkPicturePlayback.cpp
+    src/core/SkPictureRecord.cpp
+    src/core/SkPictureRecorder.cpp
+    src/core/SkPixelRef.cpp
+    src/core/SkPixmap.cpp
+    src/core/SkPixmapDraw.cpp
+    src/core/SkPoint.cpp
+    src/core/SkPoint3.cpp
+    src/core/SkPtrRecorder.cpp
+    src/core/SkQuadClipper.cpp
+    src/core/SkRRect.cpp
+    src/core/SkRSXform.cpp
+    src/core/SkRTree.cpp
+    src/core/SkRasterClip.cpp
+    src/core/SkRasterPipeline.cpp
+    src/core/SkRasterPipelineBlitter.cpp
+    src/core/SkReadBuffer.cpp
+    src/core/SkReadPixelsRec.cpp
+    src/core/SkRecord.cpp
+    src/core/SkRecordDraw.cpp
+    src/core/SkRecordOpts.cpp
+    src/core/SkRecordedDrawable.cpp
+    src/core/SkRecorder.cpp
+    src/core/SkRecords.cpp
+    src/core/SkRect.cpp
+    src/core/SkRegion.cpp
+    src/core/SkRegion_path.cpp
+    src/core/SkResourceCache.cpp
+    src/core/SkRuntimeBlender.cpp
+    src/core/SkRuntimeEffect.cpp
+    src/core/SkSLTypeShared.cpp
+    src/core/SkScalar.cpp
+    src/core/SkScalerContext.cpp
+    src/core/SkScan.cpp
+    src/core/SkScan_AAAPath.cpp
+    src/core/SkScan_AntiPath.cpp
+    src/core/SkScan_Antihair.cpp
+    src/core/SkScan_Hairline.cpp
+    src/core/SkScan_Path.cpp
+    src/core/SkSpecialImage.cpp
+    src/core/SkSpriteBlitter_ARGB32.cpp
+    src/core/SkStream.cpp
+    src/core/SkStrike.cpp
+    src/core/SkStrikeCache.cpp
+    src/core/SkStrikeSpec.cpp
+    src/core/SkString.cpp
+    src/core/SkStringUtils.cpp
+    src/core/SkStroke.cpp
+    src/core/SkStrokeRec.cpp
+    src/core/SkStrokerPriv.cpp
+    src/core/SkSwizzle.cpp
+    src/core/SkSwizzler_opts.cpp
+    src/core/SkSwizzler_opts_hsw.cpp
+    src/core/SkSwizzler_opts_ssse3.cpp
+    src/core/SkTaskGroup.cpp
+    src/core/SkTextBlob.cpp
+    src/core/SkTextBlobTrace.cpp
+    src/core/SkTypeface.cpp
+    src/core/SkTypefaceCache.cpp
+    src/core/SkTypeface_remote.cpp
+    src/core/SkUnPreMultiply.cpp
+    src/core/SkVertState.cpp
+    src/core/SkVertices.cpp
+    src/core/SkWriteBuffer.cpp
+    src/core/SkWritePixelsRec.cpp
+    src/core/SkWriter32.cpp
+    src/core/SkYUVAInfo.cpp
+    src/core/SkYUVAPixmaps.cpp
+    src/core/SkYUVMath.cpp
+    src/core/SkYUVPlanesCache.cpp
+
+    src/gpu/AtlasTypes.cpp
+    src/gpu/Blend.cpp
+    src/gpu/BlendFormula.cpp
+    src/gpu/BlurUtils.cpp
+    src/gpu/DitherUtils.cpp
+    src/gpu/PipelineUtils.cpp
+    src/gpu/RectanizerPow2.cpp
+    src/gpu/RectanizerSkyline.cpp
+    src/gpu/ResourceKey.cpp
+    src/gpu/ShaderErrorHandler.cpp
+    src/gpu/SkBackingFit.cpp
+    src/gpu/Swizzle.cpp
+    src/gpu/TiledTextureUtils.cpp
+
+    src/gpu/ganesh/ClipStack.cpp
+    src/gpu/ganesh/Device.cpp
+    src/gpu/ganesh/Device_drawTexture.cpp
+    src/gpu/ganesh/GrAttachment.cpp
+    src/gpu/ganesh/GrAuditTrail.cpp
+    src/gpu/ganesh/GrBackendSemaphore.cpp
+    src/gpu/ganesh/GrBackendSurface.cpp
+    src/gpu/ganesh/GrBackendTextureImageGenerator.cpp
```

Comment:
Do you want to move the ganesh stuff to its own variable and then when you add the other backend you could switch between the two for the sources?

Reply:
We are not completely decided on it, but for now we plan to always build Ganesh and add an option to build Graphite as well and choose at runtime among both. I understand you may want to use only Graphite for the PS and/or Win ports; so I think we can split the source list into variables when we get to that point. Does that sound good?

Reply:
Sure